Framework/PythonInterface/test/python/mantid/kernel/ArrayPropertyTest.py

```diff
@@ -7,16 +7,17 @@
                            NullValidator)
 from mantid.api import PythonAlgorithm
 import numpy as np
+import sys
 
 class ArrayPropertyTest(unittest.TestCase):
 
-    def test_default_constructor_raises_an_exception(self):
+    def xtest_default_constructor_raises_an_exception(self):
```

Comment:
Nice catch. They should be on. I'd disabled them while debugging a segfault and didn't re-enable them. I'll turn them back on.